### PR TITLE
Incorperate shared_library pip install into copy_files_to_archive

### PR DESCRIPTION
**Pull request by bhazuka**
## Summary of Changes

This is to support ORCA-205 work. This makes some minor updates to the shared libraries and adds changes needed for the copy_files_to_archive Lambda that are needed to use the shared libraries by installing them via pip.

## Changes

* Updated .gitignore so that build artifacts are ignored when installing the shared libraries
* Updated the following files in shared libraries to fix issues with version mismatches
    * README.md
    * requirements.txt
    * setup.py 
* Updated the following files in copy_files_to_archive lambda
    * `bin/build.sh` - Removed shared library copy block
    * `bin/run_tests.sh` -  Removed shared library copy block
    * `tasks/copy_files_to_archive/copy_files_to_archive.py` - Updated library import reference
    * `tasks/copy_files_to_archive/requirements.txt` and `tasks/copy_files_to_archive/requirements-dev.txt` - Added shared library install location

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Validated via run_test.sh script unit tests and performing a build and integration validation test in sandbox.

## Checklist:

- [X] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] User documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
    - [x] Manual testing/integration testing passes (if forked)
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

File: tasks/copy_files_to_archive/bin/build.sh [hunk 1]
```diff
@@ -101,24 +101,6 @@ cp ../package/awslambda-psycopg2/psycopg2-3.7/* build/psycopg2/
 let return_code=$?
 check_rc $return_code "ERROR: Unable to install psycopg2."
 
-## copy the shared_recovery.py
```
Comment (AndrewDorn) [inline tasks/copy_files_to_archive/bin/build.sh:104]: Should we remove the TODO on line 85 as well?

File: tasks/copy_files_to_archive/bin/run_tests.sh [hunk 2]
```diff
@@ -45,23 +45,7 @@ function check_rc () {
...
-let return_code=$?
-check_rc $return_code "ERROR: Unable to create orca_shared directory."
 
-touch orca_shared/__init__.py
-let return_code=$?
-check_rc $return_code "ERROR: Unable to create [orca_shared/__init__.py] file"
-
-cp ../shared_libraries/recovery/shared_recovery.py orca_shared/
-let return_code=$?
-check_rc $return_code "ERROR: Unable to copy shared library [orca_shared/shared_recovery.py]"
```
Comment (AndrewDorn) [inline tasks/copy_files_to_archive/bin/run_tests.sh:49]: Should probably remove the `rm -rf orca_shared`

File: tasks/copy_files_to_archive/copy_files_to_archive.py [hunk 3]
```diff
@@ -79,7 +79,8 @@ def task(
                     a_file[INPUT_SOURCE_BUCKET_KEY],
                     a_file[INPUT_SOURCE_KEY_KEY],
                     a_file[INPUT_TARGET_BUCKET_KEY],
-                    a_file.get(INPUT_MULTIPART_CHUNKSIZE_MB, None) or default_multipart_chunksize_mb,
+                    a_file.get(INPUT_MULTIPART_CHUNKSIZE_MB, None)
+                    or default_multipart_chunksize_mb,
```
Comment (AndrewDorn) [inline tasks/copy_files_to_archive/copy_files_to_archive.py:83]: If this needs to be on a second line, can we indent it?

**Review by AndrewDorn — COMMENTED** [review]
Found a couple non-functional issues, but overall looks good.

File: tasks/copy_files_to_archive/bin/run_tests.sh (hunk 2 above)
Comment (bhazuka) [reply tasks/copy_files_to_archive/bin/run_tests.sh:49]: yep, missed that one.

File: tasks/copy_files_to_archive/copy_files_to_archive.py (hunk 3 above)
Comment (bhazuka) [reply tasks/copy_files_to_archive/copy_files_to_archive.py:83]: Auto formatting caught  it, I can change it back to a single line

**Review by AndrewDorn — APPROVED** [review]
Don't see anything other than that todo in build.sh

**Pull request merged**